### PR TITLE
Colorize Error Message on Windows

### PR DESCRIPTION
**Pull request by dubeejw**
- Dislplay error prefix in color on Windows OS

### CODE REVIEW

**Comment by dubeejw** [comment]
PG2 279.

**Comment by dubeejw** [comment]
@mdeuser, ready for review.

File: tools/cli/go-whisk-cli/main.go [hunk 1]
```diff
@@ -79,15 +80,16 @@ func main() {
...
+            fmt.Fprintf(outputStream, "%s%s\n", color.RedString(T("error: ")), err)
         } else if displayMsg && !msgDisplayed && exitCode == 0 {
-            fmt.Fprintf(os.Stderr, "%s\n", err)
+            fmt.Fprintf(outputStream, "%s\n", err)
         }
 
-
         // Displays usage
         if displayUsage {
             fmt.Fprintf(os.Stderr, T("Run '{{.Name}} --help' for usage.\n",
```
Comment (mdeuser) [inline tools/cli/go-whisk-cli/main.go]: Just in case, use the colorable stderr here as well.

**Comment by csantanapr** [comment]
@dubeejw What's the status on this PR? Are you waiting for @mdeuser to do a second review?

**Comment by dubeejw** [comment]
@csantanapr, I made the small change that Mark pointed out. Should be ready now.

**Comment by csantanapr** [comment]
Ok removing "review" label then, and adding "ready"

**Comment by csantanapr** [comment]
rebase and merged 130e50a371ada0b3fc0930417835635b7f70adce

**Comment by mdeuser** [comment]
LGTM

**Pull request closed without merging**